assets/sparksql/ingest.py

```
@@ -0,0 +1,14 @@
+# executed within container
+import sys
+
+from pyspark.sql import SparkSession
+
+spark = SparkSession \
+    .builder \
+    .appName("") \
+    .master("spark://localhost:7077") \
+    .config("spark.sql.caseSensitive", True) \
```

Ruff (0.12.2)

Boolean positional value in function call

(FBT003)